.promu.yml

```diff
@@ -0,0 +1,37 @@
+go: 1.5.4
+repository:
+    path: github.com/prometheus/mysqld_exporter
+build:
+    flags: -a -tags netgo
+    ldflags: |
+        -X {{repoPath}}/version.Version={{.Version}}
+        -X {{repoPath}}/version.Revision={{.Revision}}
+        -X {{repoPath}}/version.Branch={{.Branch}}
+        -X {{repoPath}}/version.BuildUser={{user}}@{{host}}
+        -X {{repoPath}}/version.BuildDate={{date "20060102-15:04:05"}}
+tarball:
+    files:
+        - LICENSE
+        - NOTICE
+crossbuild:
+    platforms:
+        - linux/amd64
+        - linux/386
+        - darwin/amd64
+        - darwin/386
+        - windows/amd64
+        - windows/386
+        - freebsd/amd64
+        - freebsd/386
+        - openbsd/amd64
+        - openbsd/386
+        - netbsd/amd64
+        - netbsd/386
+        - dragonfly/amd64
+        - linux/arm
+        - linux/arm64
+        - freebsd/arm
+        - openbsd/arm
+        - netbsd/arm
+        - linux/ppc64
+        - linux/ppc64le
```

Comment:
MySQL's platform support is pretty good.  The officially supported by oracle list is fairly short, but the source will compile to pretty much anything *NIXy.